server/src/providers/blockchain.ts

```diff
@@ -430,7 +441,7 @@ export class blockchain {
         log.trace('Skipping system contracts event', { fromAddr })
       }
 
-      if (this.isClientWallet(toAddr)) {
+      if (this.isClientWallet(toAddr) && (this.isClientWallet(fromAddr) || this.isPaymentlinkContracts(fromAddr))) {
```

Comment:
What is the reason for this change?

Reply:
it was task - `change calculation of "in" txs, incoming tx is counted when the "from" is the paymentlink contracts or the from is not a system account`